packages/client/ui/react-ui/src/providers/CrossmintAuthProvider.tsx

```diff
@@ -47,12 +47,15 @@ export function CrossmintAuthProvider({ embeddedWallets, children, appearance }:
     const crossmintBaseUrl = validateApiKeyAndGetCrossmintBaseUrl(crossmint.apiKey);
     const [modalOpen, setModalOpen] = useState(false);
 
-    const setAuthMaterial = (authMaterial: AuthMaterial) => {
-        setCookie(SESSION_PREFIX, authMaterial.jwtToken);
-        setCookie(REFRESH_TOKEN_PREFIX, authMaterial.refreshToken.secret, authMaterial.refreshToken.expiresAt);
-        setJwt(authMaterial.jwtToken);
-        setRefreshToken(authMaterial.refreshToken.secret);
-    };
+    const setAuthMaterial = useCallback(
```

Comment:
same here, more for my own info
React is a bit rusty to understand the benefit of using `useCallback` here. Mind explaining? (DM best so I see faster)